app/models/chef.rb

```diff
@@ -1,4 +1,11 @@
 class Chef < ApplicationRecord
    validates_presence_of :name
    has_many :dishes
+
+   def chef_ingredients
+      self.dishes
```

Comment:
Nice work! Another solution for this story would be adding `has_many :ingredients, through: :dishes` in this model because it already has an association with dishes so you can build off of that relationship. Once it's set up, you can call ingredients directly as an attribute of a chef, i.e. `chef.ingredients.distinct`